### DISCUSSION TITLE
BTV Action

### DISCUSSION THREAD

**Discussion by dicheto** [iptv-org/community#478] [Show and tell]
### Channel ID

bTVAction.bg

### Stream URL (optional)

https://de1.glebul.com/hls/hd-btv-action-hd/index.m3u8?e=1692278132&hash=oDY7KfzmHpafrskdmaYi3w

### Notes (optional)

Is this link okay?

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://de1.glebul.com/hls/hd-btv-action-hd/index.m3u8?e=1692278132&hash=oDY7KfzmHpafrskdmaYi3w (offline)
